### PR TITLE
[Access-tokens] Backport test and dependencies from 'master'

### PR DESCRIPTION
**Pull request by jprzychodzen**
Backport:
* Access-tokens test
* Template function 'Loop'
* Template function 'ifThenElse'
* 'Sleep' measurement
* backport simple_exectuor template parameters

Ref. kubernetes/kubernetes#83375

### CODE REVIEW

File: verify/verify-flags/known-flags.txt [hunk 1]
```diff
@@ -20,6 +20,10 @@ n-runs-count
 ping-server-address
 ping-server-bind-address
 ping-sleep-duration
+prometheus-scrape-etcd
+prometheus-scrape-kube-proxy
```
Comment (mborsz) [inline verify/verify-flags/known-flags.txt]: Is this needed? You don't backport any Prometheus component?

Comment (jprzychodzen) [reply verify/verify-flags/known-flags.txt]: It's not necessary, but if someone uses `https://github.com/kubernetes/perf-tests#repository-setup` from master branch, then it breaks pushing.

**Pull request merged**